parts/k8s/windowsnodereset.ps1

```diff
@@ -45,7 +47,13 @@ if ($global:CsiProxyEnabled) {
 $hnsNetwork = Get-HnsNetwork | Where-Object Name -EQ azure
 if ($hnsNetwork) {
     Write-Log "Cleaning up containers"
-    docker ps -q | ForEach-Object { docker rm $_ -f }
+    if ($UseContainerD -eq $true) {
+        ctr -n k8s.io c ls -q | ForEach-Object { ctr -n k8s.io tasks kill $_ }
+        ctr -n k8s.io c ls -q | ForEach-Object { ctr -n k8s.io c rm $_ }
```

Comment:
Is there a -f equiv on the ctr.exe commands (like the docker calls below).
Or have you seen the tasks kill operation not complete succesfully - i'm just curious what would happen to the c rm call if the tasks kill command failed

Reply:
No there's no force option :/